.github/workflows/maven-verify.yml

```diff
@@ -272,6 +272,10 @@ jobs:
       - name: Build with Maven
         if: steps.should-run.conclusion == 'success'
         run: ./mvnw --errors --batch-mode --show-version ${{ inputs.maven_args }} ${{ inputs.verify-goal }}
+        
+      - name: Clean Ensuring no file handle remains open on windows
+        if: steps.should-run.conclusion == 'success'
+        run: ./mvnw clean        
```

Comment:
As I understand each GH job is executed in new VM which is removed after job finish, so all used resources should be decommissioned and should not have impact to other jobs.

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#using-a-github-hosted-runner

Reply:
yes for sure but this is not the goal of the check.
The goal is to be sure we do not left file for people who wants to build this locally.
if you cannot run `mvn test` then `mvn clean test` because of file leaking in a windows env we should consider this as a bug.

Reply:
I don't think that somebody use GH scripts locally ...
In GH we take additional a few seconds for build 

if needed pleas also add `--errors --batch-mode --show-version`
and add the same to `fail-fast-build`

Reply:
With locally I mean locally on your laptop :)
This check is here to detect our builds/plugins/etc do not leave files open for windows users.
It would be a problem if anything leave files open for windows users.

Reply:
I see, we want to detect in this step if all files are closed - if no - build will break